wolfssl_client_trust_platform/firmware/README.md

```diff
@@ -0,0 +1,193 @@
+# CryptoAuth Trust Platform wolfSSL Example
+
+## Hardware
+
+* DM320118: CryptoAuth Trust Platform
+* Wifi7click: ATWINC1510
+
+## Software
+
+* MPLAB X v5.40
+* Microchip XC32 v2.50 compiler
+* Microchip Harmony 3
+* cryptoauthlib: v3.2.3
+* wolfSSL: v4.5.0
+
+## Building
+
+1. Create a new project for your target board
+
+2. Using the Harmony Configuration tool make sure the following components are selected:
+
+* ATECC608
+  - Click on the Instance 0 and choose interface type -> "Trust & Go: TLS"
+* Trust&Go
+  - Choose TNGTLS Certificates and Legacy Trust Certificates.
+* wolfSSL
+* wolfCrypt
+* WINC
+
+3. Harmony v3 code generation needs a few changes
+
+See wolfssl_trust_platform_patch.diff and apply using `patch -p1 < wolfssl_trust_platform_patch.diff`.
+
+Here are the changes:
+
+* `src/config/samd21_trust/configuration.h`:
+
+```c
+#if 0 /* disabled to save code space */
+#define WOLFSSL_ALT_NAMES
+#define WOLFSSL_DER_LOAD
+#define KEEP_OUR_CERT
+#define KEEP_PEER_CERT
+#define HAVE_CRL_IO
+#define HAVE_IO_TIMEOUT
+#endif
+
+/* Disable because WINC1500 is being used */
+//#define MICROCHIP_TCPIP
```

Comment:
No, this section gets uncommented when harmony is run, so its here to clarify the line need uncommented because WINC1510 is used.

Reply:
Ok, thanks David